Python/compile.c

```diff
@@ -1528,6 +1535,11 @@ compiler_addop_j(struct compiler *c, int opcode, basicblock *b)
         return 0; \
 }
 
+#define ADDOP_JUMP_NOLINE(C, OP, O) { \
```

Comment:
Small suggestion: we could use something like `ADDOP_ARTIFICIAL_JUMP` instead of `ADDOP_JUMP_NOLINE` so is clear the kind of jump more than how it will behave

Reply:
I think I prefer `NOLINE` to `ARTIFICIAL`. `NOLINE` explains *what* it does, `ARTIFICIAL` explains *why*.
I'd prefer the code to say what it does, and leave the "why" to comments.
I'll add a comment to explain why the opcode should have no line number.

Reply:
Makes sense